react/states_and_effects/how_to_deal_with_side_effects.md

```diff
@@ -0,0 +1,236 @@
+### Introduction
+
+Certain components in react need to be interacting with other things outside the component itself. These other things can be anything from querying data from a server to synchronizing with the position of the component on the webpage or even sending some data to a server when necessary. This leads to the arising of a new concept called 'effects'. 
```

Comment:
> to synchronizing with the position of the component on the webpage

I find this vague, can you elaborate?

Reply:
I thinking it'd be better to say "to finding/changing the position of a component with respect to te rest of the webpage"?

Reply:
@01zulfi ?

Reply:
@Lofty-Brambles  Thanks for making all the changes, I'll take a look soon-ish